internal/e2e/e2eutil/compare_documents.go

```diff
@@ -99,18 +110,84 @@ func getFile(script *testscript.TestScript, filePath string) *os.File {
 	return file
 }
 
-func compareMetaData(script *testscript.TestScript, first, second *sbom.Metadata) bool {
-	firstStr := first.String()
-	secondStr := second.String()
+func compareMetaData(script *testscript.TestScript, have, want *sbom.Metadata) bool {
+	// append protobom tool entry to match expected value
+	if format := want.GetSourceData().GetFormat(); strings.Contains(format, "spdx") {
+		want.Tools = append(want.GetTools(), &sbom.Tool{Name: "protobom-devel"})
+		slices.SortFunc(want.GetTools(), func(i, j *sbom.Tool) int {
+			return strings.Compare(i.GetName(), j.GetName())
+		})
+	}
+	// remove sourceData to avoid hash mismatches due to format specific losses
+	have.SourceData = nil
+	want.SourceData = nil
+
+	// remove timestamp to ignore time created mismatch
```

Comment:
nits: casing, end comments with period, blank line after `if` block

```suggestion

	// Remove SourceData to avoid hash mismatches due to format specific losses.
	have.SourceData = nil
	want.SourceData = nil

	// Remove Date to ignore time created mismatch.
```